Library/Homebrew/formula_auditor.rb

```diff
@@ -383,7 +383,7 @@ def audit_homepage
                                                          user_agents:   [:browser, :default],
                                                          check_content: true,
                                                          strict:        @strict))
-        problem http_content_problem
+        problem http_content_problem.sub("URL", "homepage URL")
```

Comment:
I think ideally `homepage URL` would be an argument to `curl_check_http_content` rather than substituting the output at the end. It would be good to make this a required argument and also use it in `Library/Homebrew/cask/audit.rb` too.

Reply:
That makes sense! I’ll also try to look into what I can do for this tonight.